ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_filter.proto

```diff
@@ -70,10 +84,25 @@ message InterfaceFilter {
   // Optional
   bool include_interface_view = 2;
 
-  // Whether to include a ``create_arguments_blob`` in the returned
+  // Whether to include a ``contract_create_payload`` in the returned
+  // ``CreateEvent``.
+  // Use this to access the complete contract payload in your API client
+  // for submitting it as a disclosed contract with future commands.
+  // Optional
+  bool include_contract_create_payload = 3;
```

Comment:
this name is now inconsistent with the one in the `InterfaceFilter`.

As suggested above: I'd minimize the polish, and just use the same name here as for `InterfaceFilters`.

Reply:
Same comment as for the command.proto

Reply:
The change of the blog type from `Any` -> `bytes` has precipitated some further changes. So this comment no longer stands